src/main/java/org/terasology/weatherManager/systems/EmitWeatherParticleSystem.java

```diff
@@ -80,51 +76,40 @@
     private int minDownfall;
     private int maxDownfall;
 
-    @ReceiveEvent
-    public void playerSpawned(OnPlayerSpawnedEvent event, EntityRef player) {
-        begin(player);
-    }
-
-    @ReceiveEvent
-    public void playerRespawned(OnPlayerRespawnedEvent event, EntityRef player) {
-        begin(player);
-    }
-
     @ReceiveEvent(components = ClientComponent.class)
     public void onConnect(ConnectedEvent connected, EntityRef entity) {
-        begin(entity);
+        if (localPlayer.getId() == 0)
+            localPlayer = entity;
```

Comment:
We should get some style checker running for modules again 🧐 please do _always_ use braces with if-statements. 
```suggestion
        if (localPlayer.getId() == 0) {
            localPlayer = entity;
        }
```

Reply:
same for the other occurrences

Reply:
alright